deployments/kakarot-sepolia/declarations.json

```diff
@@ -1,9 +1,9 @@
 {
-  "kakarot": "0x5fa636d9904ec7b1ac2c50ff36ccbe3d38db666b7fafd4107c68c8333bf0bec",
-  "account_contract": "0x490cccb64e3917ecf0a80a59d0c3e449766f745c1d6db54e0050f89eb59aba1",
-  "account_contract_fixture": "0x2c2520d81c315969597b2f7a0189db1b65ec19da02fe40c40d183e2e9bfb3c8",
+  "kakarot": "0x7e5e97d10a2db3957687ba0a4bb5afde05bf7468871514af416d70badb711f8",
+  "account_contract": "0x3c1081f98ecdd1d320368d2a136191a3fbf02a3e419463d77c3c5aa6dbc99a0",
+  "account_contract_fixture": "0x5e823aa6a4468e35903689615b834fe280a1f6a039b22d0d577f551a50cda03",
```

Comment:
`account_contract_fixture` `EVM` `Cairo1HelpersFixture` are not meant to be declared on prod networks, they're only classes used for tests

Reply:
why were they declared?

Reply:
idk, shouldnt be on testnets, only devnets

Reply:
because well, currently there is nothing in the `DECLARED_CONTRACTS` constant to give such information!

You could add a `'only_devnet': bool` in (https://github.com/ClementWalter/kakarot/blob/98b26fda32c36f09880ed0c7f44dba7f4d669b61/kakarot_scripts/constants.py#L185)

and filter in https://github.com/ClementWalter/kakarot/blob/98b26fda32c36f09880ed0c7f44dba7f4d669b61/kakarot_scripts/deploy_kakarot.py#L38

Reply:
note that it's not been done yet because it was especially made it possible to run the end-to-end tests against any RPC (e.g. starknet testnet), but it's not been used so far